src/parseOption.ts

```diff
@@ -41,6 +41,18 @@ export function getTooltipData(item: Scenegraph, options: Option) {
   } else {
     tooltipData = prepareCustomFieldsData(itemData, options);
   }
+
+  if (options.sort) {
+    const sortStr =
+      options.sort === 'title' ? 'title' :
+      options.sort === 'value' ? 'rawValue' :
+      null;
+    const sortFn = sortStr ? defaultSort(sortStr) : typeof options.sort === 'function' && options.sort;
```

Comment:
You can use `isFunction` and all other `vega-util` function to check object type.